apiserver/facades/client/applicationoffers/state.go

```diff
@@ -106,6 +110,14 @@ func (s *stateShim) Model() (Model, error) {
 	return &modelShim{m}, err
 }
 
+func (s *stateShim) GetModelCallContext() (context.ProviderCallContext, error) {
+	m, err := s.st.Model()
```

Comment:
I don't get this. m.State() is the same as s.st so getting the model is redundant.

Reply:
True, changed to s.st